src/OpenRiaServices.Client.DomainClients.Http/Framework/Http/BinaryHttpDomainClient.cs

```diff
@@ -193,11 +194,60 @@ async Task<QueryCompletedResult> QueryAsyncCoreContinuation()
         /// <param name="cancellationToken"></param>
         private Task<HttpResponseMessage> PostAsync(string operationName, IDictionary<string, object> parameters, List<ServiceQueryPart> queryOptions, CancellationToken cancellationToken)
```

Comment:
Should we add some tests?

Reply:
Would be nice to not have to do manual testing. this almost just a copy paste of old code. I hve a separate tracking issue to run all integration tests in ci build https://github.com/OpenRIAServices/OpenRiaServices/issues/359